cmake/Modules/LibFetchMacros.cmake

```
@@ -0,0 +1,60 @@
+include(FetchContent)
```

It's a best practice to set minimum versions for scripts. Since we're using `FetchContent` the minimum version should be 3.11.
```suggestion
cmake_minimum_required(VERSION 3.11)
include(FetchContent)
```